src/server.c

```diff
@@ -4328,6 +4325,7 @@ sds genRedisInfoString(const char *section) {
             "process_supervised:%s\r\n"
             "run_id:%s\r\n"
             "tcp_port:%i\r\n"
+            "server_time_usec:%I.%I\r\n"
```

Comment:
We have 3 options:
1) print this one using a separate call to sdscatprintf 
2) improve sdscatfmt so that it's able to pad zeros
3) just print usec, and let the user do the division on his side. (i vote for this one)
```suggestion
            "server_time_usec:%I\r\n"
```